Tests/ContentstackTests.swift

```diff
@@ -49,11 +49,6 @@ final class ContentstackTests: XCTestCase {
         XCTAssertEqual(stack.host, host)
     }
 
-    func testStack_Branch_ReturnStackWithBranch () {
```

Comment:
Could you please clarify the reason for removing the test cases block?

Reply:
This is outdated code @sunil-lakshman . This needs to run in the main thread else an error is thrown. So will revisit it later.